brotli/broti.go

```diff
@@ -0,0 +1,5 @@
+package broti
+
+import (
+	"C"
+)
```

Comment:
new line is missing

Reply:
Okay. I cannot push to the base repository of this PR, so I've forked it and opened https://github.com/cloudflare/brotli-go/pull/3